src/components/shared/widget.js

```diff
@@ -1,105 +1,108 @@
-import React from 'react';
-import { graphql } from 'gatsby';
-import Accordion from 'components/shared/accordion';
-import BlockWidget from 'components/shared/blockWidget';
-import GeneralText from 'components/shared/generalText';
-import ImageOverlay from 'components/shared/imageOverlay';
-import LeadPara from 'components/shared/leadPara';
-import LinksItems from 'components/shared/linksItems';
-import MediaText from 'components/shared/mediaText';
-import PageContainer from 'components/shared/pageContainer';
-import ModalVideo from 'components/shared/modalVideo';
-import PageTabs from 'components/shared/pageTabs';
-import SectionWidgets from 'components/shared/sectionWidgets';
-import StatisticWidget from 'components/shared/statisticWidget';
-import StatsWidget from 'components/shared/statsWidget';
-import Story from 'components/shared/story';
-import TestimonialSlider from 'components/shared/testimonialSlider';
-import YamlWidget from 'components/shared/yamlWidget';
-import { ConditionalWrapper, slugify } from 'utils/ug-utils';
+import React from "react";
+import { graphql } from "gatsby";
+import Accordion from "components/shared/accordion";
+import BlockWidget from "components/shared/blockWidget";
+import GeneralText from "components/shared/generalText";
+import ImageOverlay from "components/shared/imageOverlay";
+import LeadPara from "components/shared/leadPara";
+import MediaText from "components/shared/mediaText";
+import PageContainer from "components/shared/pageContainer";
+import ModalVideo from "components/shared/modalVideo";
+import PageTabs from "components/shared/pageTabs";
+import SectionWidgets from "components/shared/sectionWidgets";
+import StatisticWidget from "components/shared/statisticWidget";
+import StatsWidget from "components/shared/statsWidget";
+import Story from "components/shared/story";
+import TestimonialSlider from "components/shared/testimonialSlider";
+import YamlWidget from "components/shared/yamlWidget";
+import { ConditionalWrapper, slugify } from "utils/ug-utils";
+import LinksWidget from "./linksWidget";
```

Comment:
Minor tweak: move this import up the list to preserve alphabetical order, i.e. under `LeadPara`